pagopa/io-app
# chore: [IOBP-620] Add `ONBOARDABLE` filter to onboarding payment methods

> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/5674, https://github.com/pagopa/io-dev-api-server/pull/363

## Short description
This PR adds the `ONBOARDABLE` filter for the payment methods usable for the onboarding

## List of changes proposed in this pull request
- Updated pagoPA wallet and ecommerce API definitions
- Added `ONBOARDABLE` filter to the payment methods selector

## How to test
In an UAT environment, start a payment method onboarding flow. In the payment method selection screen you should be able to see only `ONBOARDABLE` methods
